src/Akeneo/Connectivity/Connection/front/src/connect/components/ActivateAppButton.tsx

```diff
@@ -4,7 +4,7 @@ import {useTranslate} from '../../shared/translate';
 import {Button} from 'akeneo-design-system';
 import {useRouter} from '../../shared/router/use-router';
 
-export const ActivateAppButton: FC<{id: string}> = ({id}) => {
+export const ActivateAppButton: FC<{id: string; connected: boolean}> = ({id, connected}) => {
```

Comment:
```suggestion
export const ActivateAppButton: FC<{id: string; isConnected: boolean}> = ({id, isConnected}) => {
```
to be consistent with isAuthorized

Reply:
it's modified, good idea :)